Core/Core/Providers/Ajax/AjaxProvider.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  AjaxProvider.swift
+//  Core
+//
+//  Created by Eugene Yatsenko on 12.12.2023.
+//
+
+import Foundation
+import WebKit
+import Swinject
+
+public extension NSNotification {
+    static let blockCompletion = Notification.Name.init("block_completion")
+}
```

Comment:
Notifications are centralized in `Core/Notification.swift`. Can you please move this notification to there as well.